adapters/compound_adapter.py

```diff
@@ -40,6 +40,7 @@ class CompoundAdapter(Adapter):
 
     def __init__(self, filepath=None, label="compound", dry_run=False):
         self.label = label
+        self.file_path = None
```

Comment:
This line here is redundant as it is doing the same thing as line 57

Reply:
> This line here is redundant as it is doing the same thing as line 57

yes but since i was trying to access the compound instance on the has component adapter, i needed to define the variable. since i wrote independent adapter now, i'll delete that.